pkg/otlp/model/attributes/attributes.go

```diff
@@ -174,19 +173,14 @@ func OriginIDFromAttributes(attrs pcommon.Map) (originID string) {
 
 // ContainerTagFromAttributes extracts the value of _dd.tags.container from the given
 // set of attributes.
-func ContainerTagFromAttributes(attr map[string]string) string {
-	var str strings.Builder
+func ContainerTagFromAttributes(attr map[string]string) map[string]string {
```

Comment:
Once this is merged can you bump the `pkg/otlp/model` module on the Datadog exporter? This breaks the public API

Reply:
Sure.